### ISSUE TITLE
We shouldn't delete fontinfo.plist / openTypeOS2Type

### ISSUE THREAD

**Issue by bobh0303** [silnrsi/pysilfont#52]
Regarding this line:

https://github.com/silnrsi/pysilfont/blob/6f1f79a9dac4289320e60964c6b72ade64742422/lib/silfont/scripts/psfglyphs2ufo.py#L43

With `openTypeOS2Type` key absent from the ufo, `ufo2ft` (used by `psfufo2ttf`) defaults to _Restricted License Embedding_, as [indicated here](https://github.com/googlei18n/ufo2ft/blob/87e858d5d2cb3ee7ebcf39632b06afad8bfa9c25/Lib/ufo2ft/fontInfoData.py#L351).

SIL best practice is to have _Installable Embedding_ (OS2.fsType = 0 in the TTF) and to do that we need the following in the UFO:
```
  <key>openTypeOS2Type</key>
  <array/>
```

**Comment by bobh0303** [comment]
I suggest that both `psfglyphs2ufo` and `ufo.py` check/fix routines should add the key to the UFO if it is missing.

**Comment by DavidRaymond** [comment]
Fixed in https://github.com/silnrsi/pysilfont/commit/2d162a46e954daccd63d76afb4716ad4c355953b

**Comment by bobh0303** [comment]
Hm. Maybe we still have an issue -- at least for SIL workflow.

First, just to confirm that if, in GlypshApp, I set the `fsType` custom property to the default "Installable" value, then it roundtrips to/from ufo just fine. So that much is working.

However, if a font is created in GlyphsApp, saved without setting the `fsType` custom property, and then converted to UFO using psfglyphs2ufo, the resultant fontinfo.plist will have restrictive embedding, which goes against our SIL best practice. However, `check` and `fix` modes neither complain about nor fix this (which means it is doing exactly what I proposed above when I recommended to "set to empty array _if missing_").

But I'm re-thinking that now. I now propose (and am willing to be shot down):
- in `check` mode we warn if it is missing or if it is present but not an empty array 
- in `fix` mode we force it to be an empty array.
